### ISSUE TITLE
Latest Build Issues

### ISSUE THREAD

**Issue by shad0whawk** [anthonya1999/GoodNight#57]
I'm using the latest build as four hours ago.  Two questions.  (1) If I enable darkroom mode, previously it was red on black, now it looks almost exactly like turning on the "invert colors" option on the accessibility menu.  Is that the intent?  (2) If I enable darkroom mode, then select "reset brightness", the darkroom mode toggle is disabled but the screen doesn't change back.  I have to reenable the toggle and then disable it again to revert to normal.

**Comment by anthonya1999** [comment]
@shad0whawk It was actually a bug before with the red on black. It is supposed to be like the "invert colors function. And the second issue will be fixed in a second...

**Comment by mariokorte** [comment]
Hm, the red on black was intentionally implemented that way - just like the darkroom mode in f.lux.
I would like to revert it to that state, as it is way darker. If you lay at night in bed with your spouse and cannot sleep and therefore look into your smartphone the new mode is way too dark for me (and my wife) ;)
Suggestions welcome as to let it as is (invert colors) or revert to real darkroom mode.

**Comment by mariokorte** [comment]
OK, did check back with @anthonya1999 . Will revert to previous functionality soon.

**Comment by mariokorte** [comment]
Closed with #60
